lib/features/temp/presentation/screens/temp/temp_model.dart

```diff
@@ -1,34 +1,30 @@
-import 'package:elementary/elementary.dart';
 import 'package:flutter_template/common/service/theme/theme_service.dart';
 import 'package:flutter_template/config/environment/environment.dart';
+import 'package:flutter_template/core/architecture/presentation/base_model.dart';
 import 'package:flutter_template/features/temp/domain/repositories/i_temp_repository.dart';
 import 'package:flutter_template/features/temp/presentation/screens/temp/temp_screen.dart';
 
 /// Model for [TempScreen].
-final class TempModel extends ElementaryModel implements ITempModel {
+final class TempModel extends BaseModel {
```

Comment:
Do you think interface for ElementaryModel is a bad idea?

Reply:
It is a good idea, but it is not consistent for the current state of the template. I think, we can add this improvement in a separate pr.